src/docfx/docset/Docset.cs

```diff
@@ -34,11 +40,13 @@ internal class Docset
 
         private readonly CommandLineOptions _options;
         private Lazy<Dictionary<string, Docset>> _dependentDocsets;
+        private Lazy<Dictionary<string, string>> _redirections;
 
         public Docset(string docsetPath, CommandLineOptions options)
             : this(docsetPath, Config.Load(docsetPath, options), options)
         {
             _dependentDocsets = new Lazy<Dictionary<string, Docset>>(() => LoadDependencies());
+            _redirections = new Lazy<Dictionary<string, string>>(() => LoadRedirectionMappings(docsetPath, "docfx-redirect.yml"));
```

Comment:
>LoadRedirectionMappings [](start = 71, length = 23)

Add a `redirections` property to Config object. We should use the `extend` syntax to include other files.